src/Ocelot/DependencyInjection/ConfigurationBuilderExtensions.cs

```diff
@@ -32,12 +29,12 @@ public static IConfigurationBuilder AddOcelotBaseUrl(this IConfigurationBuilder
             return builder;
         }
 
-        public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder, IWebHostEnvironment env)
+        public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder, IHostEnvironment env)

```

Comment:
You are proposing breaking changes!
You must add overloaded version of the method!